keycloak-iam/src/main/java/com/valtimo/keycloak/service/KeycloakService.java

```diff
@@ -62,6 +62,9 @@ public RolesResource clientRolesResource() {
     }
 
     public String getClientId() {
+        if (clientName.isBlank()) {
```

Comment:
By the looks of it, clientName can also be `null`, which will throw a NPE

Reply:
No it can't by null. This is how the clientName is checked in all other places

Reply:
I disagree on the fact that we're relying on other classes/configuration to ensure null-safety in this one. It would be better to check for nulls in the constructor. 

However, the result would be the same: some exception would be thrown at runtime, so I'm letting it pass.